client/components/description.jsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import BoatDescription from './boatDescription.jsx'
+import dummy from './../../mockData.js';
 
 class Description extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      boat: dummy[0];
+    }
+  }
 
+  render() {
+    return {
```

Comment:
minor thing - should the return statement have parentheses around the jsx instead of curly braces, i.e. 
```
return (
  <div>
    ...
  </div>
)
```
or does that not actually matter?

Reply:
I am not sure which way is preferred, but using () on an arrow function gives it an implicit return so if I used {} I would have to right "return" in front of the first div